app/components/organization-team/invitation-list/index.hbs

```diff
@@ -0,0 +1,66 @@
+<AkStack @alignItems='center' @justifyContent='space-between' class='mb-3'>
+  <AkStack @direction='column' @spacing='0.5'>
+    <AkTypography data-test-teamInviteList-title @variant='h6'>
+      {{t 'teamInvitations'}}
+    </AkTypography>
+
+    <AkTypography
+      data-test-teamInviteList-description
+      @color='textSecondary'
+      @variant='body2'
+    >
+      {{t 'teamInvitationsDesc'}}
+    </AkTypography>
+  </AkStack>
+
+  <AkStack @spacing='1'>
+    {{!-- <div {{style width='180px'}}>
```

Comment:
Commented code?

Reply:
at the api does not support search so will uncomment once backend change is there